Portfolio/index.html

```diff
@@ -1,16 +1,116 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
```

Comment:
The 'top' attribute is not a valid attribute for a div tag. It should be removed.